src/api/app/views/webui/users/tokens/show.html.haml

```diff
@@ -0,0 +1,60 @@
+- @pagetitle = @token.name.presence || 'No name'
```

Comment:
If multiple tabs are open, this small change helps in telling what the tab is about:

```suggestion
- @pagetitle = "Token - #{@token.name.presence || 'No name'}"
```